packages/usa-in-page-navigation/src/index.js

```diff
@@ -51,12 +53,20 @@ const setActive = (el) => {
 /**
  * Return a node list of section headings
  *
+ * @param {String} headings A string of one or more headings to query for
  * @return {HTMLElement[]} - An array of DOM nodes
  */
-const getSectionHeadings = () => {
-  const sectionHeadings = document.querySelectorAll(
-    `${MAIN_ELEMENT} h2, ${MAIN_ELEMENT} h3`
-  );
+const getSectionHeadings = (headings) => {
+  const headingList = headings.indexOf(" ") ? headings.split(" ") : headings;
```

Comment:
Is this `headings.indexOf(" ")` to check for multiple values?

Reply:
Should we validate any of the values passed into `data-headings`? Or at least return an error if a user tries to pass an invalid header _ex: j1 j2 instead of h1 h2_ ?

For example, validate against a list of expected values? Could be a `const VALID_HEADINGS = ["h1", "h2", "h3", "h4", "h5", "h6"]`

Reply:
Done! Outstanding suggestion, made it so, and it throws an error like the screen shot below if they use something other than a valid heading tag:

<img width="499" alt="image" src="https://user-images.githubusercontent.com/19338309/201200353-b61473fb-a108-412f-808c-91dd062f9eec.png">

Reply:
> Is this `headings.indexOf(" ")` to check for multiple values?

Yes, I know they could try to use some other delimiter like a comma, but was thinking we could detail this in the guidance... And with the valid headings check implemented, we should be good? What do you think?

Reply:
That might not be necessary, but that's a good question. I feel like it's good enough to throw an error once the expected heading element is invalid.

FWIW; I had thought something similar like passing a value `[h1, h2, h3]` to the data-attribute for clarity. Unfortunately, it doesn't make things any easier for anyone.